go/test/endtoend/recovery/pitr/shardedpitr_test.go

```diff
@@ -0,0 +1,540 @@
+package pitr
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"vitess.io/vitess/go/mysql"
+	"vitess.io/vitess/go/test/endtoend/cluster"
+	"vitess.io/vitess/go/vt/log"
+)
+
+var (
+	createTable = `create table product (id bigint(20) primary key, name char(10), created bigint(20));`
+	insertTable = `insert into product (id, name, created) values(%d, '%s', unix_timestamp());`
+	getCountID  = `select count(*) from product`
+)
+
+var (
+	clusterInstance *cluster.LocalProcessCluster
+
+	master        *cluster.Vttablet
+	replica       *cluster.Vttablet
+	shard0Master  *cluster.Vttablet
+	shard0Replica *cluster.Vttablet
+	shard1Master  *cluster.Vttablet
+	shard1Replica *cluster.Vttablet
+
+	cell           = "zone1"
+	hostname       = "localhost"
+	binlogHost     = "127.0.0.1"
+	keyspaceName   = "ks"
+	restoreKSName  = "restoreks"
+	restoreKS2Name = "restoreks1"
+	restoreKS3Name = "restoreks2"
```

Comment:
Make the names consistent,
```suggestion
	restoreKS1Name = "restoreks1"
	restoreKS2Name = "restoreks2"
	restoreKS3Name = "restoreks3"
```

Reply:
Done